src/Lykke.Blockchains.Integrations.Contract/Common/Responses/BlockchainErrorResponse.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+
+namespace Lykke.Blockchains.Integrations.Contract.Common.Responses
+{
+    /// <inheritdoc />
+    /// <typeparam name="TErrorCode">Type of the error code. Should be enum</typeparam>
+    [PublicAPI]
+    public class BlockchainErrorResponse<TErrorCode> : BlockchainErrorResponse
```

Comment:
Since C# 7.3 (released at least 8 month ago) [System.Enum constraint](https://blogs.msdn.microsoft.com/seteplia/2018/06/12/dissecting-new-generics-constraints-in-c-7-3/) is available for generics.

Reply:
fixed